README.md

```diff
@@ -151,7 +151,39 @@ VBoxManage storagectl `VBoxManage list vms |grep ceph-ansible_osd0|awk '{print $
 
 then run `vagrant up --provider=virtualbox` again.
 
-
+## Setup for Vagrant using OpenStack provider
+
+Install the Vagrant plugin for the openstack provider: vagrant plugin install vagrant-openstack-provider.
+
+cp site.yml.sample site.yml
```

Comment:
Made recommended changes.

----- Original Message -----
From: "Leseb" <notifications@github.com>
To: "ceph/ceph-ansible" <ceph-ansible@noreply.github.com>
Cc: "Jim Curtis" <jim.curtis@redhat.com>
Sent: Sunday, February 21, 2016 9:08:19 AM
Subject: Re: [ceph-ansible] Changes to allow ceph-ansible and vagrant to work on Openstack VMs (#518)

> @@ -151,7 +151,39 @@ VBoxManage storagectl `VBoxManage list vms |grep ceph-ansible_osd0|awk '{print $
>  
>  then run `vagrant up --provider=virtualbox` again.
>  
> -
> +## Setup for Vagrant using OpenStack provider
> +
> +Install the Vagrant plugin for the openstack provider: vagrant plugin install vagrant-openstack-provider.
> +
> +cp site.yml.sample site.yml

Make this a code block please using backquotes.

---
Reply to this email directly or view it on GitHub:
https://github.com/ceph/ceph-ansible/pull/518/files#r53569509